exporter/metric/prometheus/prometheus.go

```diff
@@ -39,11 +35,11 @@ type Exporter struct {
 	registerer prometheus.Registerer
 	gatherer   prometheus.Gatherer
 
-	counters           counters
-	gauges             gauges
-	histograms         histograms
-	summaries          summaries
-	measureAggregation MeasureAggregation
+	snapshot export.CheckpointSet
+	OnError  func(error)
+
+	DefaultHistogramBuckets  []float64
```

Comment:
maybe remove this for now until Histogram is supported.

Reply:
Nice catch, removed it.